xDrip/Scenes/Settings/BaseSettings/Cells/BaseSettingsTextInputTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  BaseSettingsTextInputTableViewCell.swift
+//  xDrip
+//
+//  Created by Ivan Skoryk on 11.04.2020.
+//  Copyright © 2020 Faifly. All rights reserved.
+//
+
+import UIKit
+
+class BaseSettingsTextInputTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var mainTextLabel: UILabel!
+    @IBOutlet weak var textField: UITextField!
+    
+    private var textChangeHandler: ((String?) -> Void)?
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        textField.delegate = self
+    }
+    
+    func configurate(mainText: String, detailText: String?, textChangeHandler: ((String?) -> Void)?) {
+        mainTextLabel.text = mainText
+        textField.placeholder = detailText
+        self.textChangeHandler = textChangeHandler
+    }
+}
+
+extension BaseSettingsTextInputTableViewCell: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
```

Comment:
I think calling `textChangeHandler` in this method is not the best option, because user might not tap `Done` button and just exit the screen. Use `IBAction` `editingChanged` for this.